pkg/registry/core/pod/rest/subresources.go

```diff
@@ -193,7 +193,7 @@ func (r *PortForwardREST) Connect(ctx context.Context, name string, opts runtime
 
 func newThrottledUpgradeAwareProxyHandler(location *url.URL, transport http.RoundTripper, wrapTransport, upgradeRequired, interceptRedirects bool, responder rest.Responder) *proxy.UpgradeAwareHandler {
 	handler := proxy.NewUpgradeAwareHandler(location, transport, wrapTransport, upgradeRequired, proxy.NewErrorResponder(responder))
-	handler.InterceptRedirects = interceptRedirects && utilfeature.DefaultFeatureGate.Enabled(genericfeatures.StreamingProxyRedirects)
+	handler.InterceptRedirects = interceptRedirects
```

Comment:
Shouldn't handler.InterceptRedirects be always false if `StreamingProxyRedirects` is not supported anymore?

Reply:
After re-read the KEP, I think you are right. It should be always false here.